src/OpenstoreSchema/Core/Entity/ProductTranslation.php

```diff
@@ -95,6 +95,11 @@ class ProductTranslation
      */
     private $usp;
 
+    /**
+     * @ORM\Column(type="string", length=2000, nullable=true, options={"comment" = "Product marketing text"})
+     */
+    private $marketing_text;
```

Comment:
@lgillie 

IMO `marketing_text` is not an ideal name... Description is already taken... Do you have an alternative name that does no convey the *department* originating request.  Is it a `short_description` ? That could help to stay generic... Any idea or marketing_text is good for you ?

Reply:
'short_description' could be recyclable in other contexts too ;)